sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -2578,4 +2578,45 @@ class DataFrameSuite extends QueryTest with SharedSQLContext {
         Row ("abc", 1))
     }
   }
+
+  test("SPARK-25860: Replace Literal(null, _) with FalseLiteral whenever possible") {
+
+    def checkPlanIsEmptyLocalScan(df: DataFrame): Unit = df.queryExecution.executedPlan match {
```

Comment:
this assumes we run `ConvertToLocalRelation`, let's use `withSQLConf` to make sure this rule is on.

Reply:
Do we actually have a way to enable/disable `ConvertToLocalRelation`?

Reply:
I see, thanks.

So, you mean to use `withSQLConf(SQLConf.OPTIMIZER_EXCLUDED_RULES.key -> "") {...}` to ensure that `ConvertToLocalRelation` is not excluded?